tests/cpp/operator/mkldnn_operator_test.cc

```diff
@@ -719,7 +707,10 @@ void TestOpEx(const OpAttrs &forward_attrs, const OpAttrs &backwards_attrs) {
             Context(), forward_attrs.attrs, inputs, ex_outputs, req,
             DispatchMode::kFComputeEx, mxnet::OpStatePtr());
         Engine::Get()->WaitForAll();
-        AssertEqual(outputs, ex_outputs);
+        // TODO(unassigned): Need to fix op, should work for the whole vector
```

Comment:
Can you assign TODO here? otherwise I am afraid it might remain TODO forever :)

Reply:
i didn't know who to assign it to. I have assigned to @pengzhao-intel for now, please feel free to suggest other names.